kclvm/sema/src/resolver/doc.rs

```diff
@@ -40,10 +40,10 @@ fn clean_doc(doc: &mut String) {
             .unwrap_or(0);
 
         lines[1..].iter_mut().for_each(|line| {
-            *line = if line.len() > 0 {
+            *line = if line.trim().len() > 0 {
```

Comment:
I think we should check the `margin` index out of bound and use the `line.chars()` instead of the slice  expression`&line[margin..]` including other UTF-8 chars.